internal/pkg/plugin/manager.go

```diff
@@ -109,6 +120,15 @@ func (manager *Manager) _ReloadConfig(newConfig config.Config) error {
 }
 
 func (manager *Manager) _RegisterShutdownSignalHandlers() {
+
```

Comment:
let's move the variable declaration of ProfileCleaner here so that it's local to the `_RegisterShutdownSignalHandlers` method:
```
var ProfileCleaner interface {
	Stop()
}
```